src/jetpack-editor-setup.js

```diff
@@ -31,7 +32,8 @@ const supportedJetpackBlocks = {
 		available: __DEV__,
 	},
 	'videopress/video': {
-		available: __DEV__,
+		// VideoPress block will be released first on iOS.
+		available: Platform.isIOS,
```

Comment:
Should we also keep this available on `__DEV__` to make it easier to work on Android locally?

Reply:
Good point. I'll update it to use `__DEV__` on Android 👍 .

Reply:
This has been fixed in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5692/commits/ceee1a8398b989f74c3663f538d25eea577f77c3.

Reply:
Note that I've reverted the update in the unit tests (https://github.com/wordpress-mobile/gutenberg-mobile/pull/5692/commits/19e0db1399f7115ea183d546409b6ef9e9281996). It's no longer needed as the block will be available on both platforms in the test environment.